tests/unit/test-class-wc-payments-account.php

```diff
@@ -3174,6 +3174,73 @@ public function test_get_tracking_info() {
 		$this->assertSame( $expected, $this->wcpay_account->get_tracking_info() );
 	}
 
+	public function test_get_recommended_payment_methods_unsupported_country() {
+		$this->assertSame( [], $this->wcpay_account->get_recommended_payment_methods( 'unsupported_country' ) );
+	}
+
+
+	public function get_recommended_payment_methods_provider() {
+		return [
+			'No PMs suggested'                  => [ 'US', [], [] ],
+			'Invalid PMs array'                 => [ 'US', [ 'test' => 'test' ], [] ],
+			'Enabled flag and priority not set' => [
+				'US',
+				[
+					[
+						'id'    => 1,
+						'title' => 'test PM',
+						'type'  => 'available',
+					],
+				],
+				[
+					[
+						'id'       => 1,
+						'title'    => 'test PM',
+						'type'     => 'available',
+						'enabled'  => false,
+						'priority' => 0,
+					],
+				],
+			],
+			'Enabled flag and priority set'     => [
+				'US',
+				[
+					[
+						'id'       => 1,
+						'title'    => 'test PM',
+						'type'     => 'available',
+						'enabled'  => true,
+						'priority' => 1,
+					],
+				],
+				[
+					[
+						'id'       => 1,
+						'title'    => 'test PM',
+						'type'     => 'available',
+						'enabled'  => true,
+						'priority' => 1,
+					],
+				],
+			],
+		];
+	}
+
+	/**
+	 * @dataProvider get_recommended_payment_methods_provider
+	 */
+	public function test_get_recommended_payment_methods_xxx( $country_code, $recommended_pms, $expected ) {
```

Comment:
Let's not give the wrong impression with stuff like `_xxx` 😉

Reply:
changed in [3af3993](https://github.com/Automattic/woocommerce-payments/pull/9825/commits/3af399332f1bc9159887a0f09db97ea47fa75fcb) . thx